helm/alfresco-content-services/templates/_helpers-url.tpl

```diff
@@ -0,0 +1,27 @@
+{{- define "alf.protocol" -}}
+{{ tpl (.Values.externalProtocol | default "http") $ }}
```

Comment:
Do you think we really need to use `tpl`  function? It looks to me all we do here is regular value expansion.